include/vkfw/vkfw.hpp

```diff
@@ -3130,8 +3130,11 @@ namespace VKFW_NAMESPACE {
     glfwCreateWindowSurface(instance, window, allocator, &output);
 
     vk::Optional<vk::AllocationCallbacks const> vulkan_hpp_allocator = nullptr;
-    if (allocator)
-      vulkan_hpp_allocator = vk::AllocationCallbacks(*allocator);
+    if (allocator) {
+      vk::AllocationCallbacks callbacks;
+      callbacks = *allocator;
+      vulkan_hpp_allocator = callbacks;
+    }

```

Comment:
Can you please wrap this into
```
#if !defined( VULKAN_HPP_NO_CONSTRUCTORS ) && !defined( VULKAN_HPP_NO_STRUCT_CONSTRUCTORS )
```
ensuring the behavior is unchanged in the general case?